typescript/packages/fireblocks/src/__tests__/fireblocks-signer.integration.test.ts

```diff
@@ -0,0 +1,72 @@
+import {
+    appendTransactionMessageInstructions,
+    createTransactionMessage,
+    pipe,
+    setTransactionMessageFeePayerSigner,
+    setTransactionMessageLifetimeUsingBlockhash,
+    signTransactionMessageWithSigners,
+} from '@solana/kit';
+import { getAddMemoInstruction } from '@solana-program/memo';
+import { getRpcBlockhash } from '@solana-keychain/test-utils';
+import { config } from 'dotenv';
+import { describe, expect, it } from 'vitest';
+
+import { FireblocksSigner } from '../fireblocks-signer.js';
+
+config();
+
+const REQUIRED_ENV_VARS = ['FIREBLOCKS_API_KEY', 'FIREBLOCKS_PRIVATE_KEY_PEM', 'FIREBLOCKS_VAULT_ACCOUNT_ID'];
+
+function hasRequiredEnvVars(): boolean {
+    return REQUIRED_ENV_VARS.every(v => process.env[v]);
+}
+
+async function createFireblocksSigner(): Promise<FireblocksSigner> {
+    const signer = new FireblocksSigner({
+        apiKey: process.env.FIREBLOCKS_API_KEY!,
+        assetId: process.env.FIREBLOCKS_ASSET_ID ?? 'SOL_TEST',
+        privateKeyPem: process.env.FIREBLOCKS_PRIVATE_KEY_PEM!,
+        useProgramCall: true,
+        vaultAccountId: process.env.FIREBLOCKS_VAULT_ACCOUNT_ID!,
+    });
+    await signer.init();
+    return signer;
+}
+
+describe('FireblocksSigner Integration', () => {
+    it.skipIf(!hasRequiredEnvVars())(
+        'signs transactions with PROGRAM_CALL',
+        async () => {
+            const signer = await createFireblocksSigner();
+            const rpcUrl = process.env.SOLANA_RPC_URL ?? 'https://api.devnet.solana.com';
+
+            // Get real blockhash from devnet
+            const { blockhash, lastValidBlockHeight } = await getRpcBlockhash(rpcUrl);
```

Comment:
IMO Just use RPC here and then remove the new `getRpcBlockhash`/util file.

```suggestion
    const rpc = createSolanaRpc(rpcUrl);
    const { value: { blockhash, lastValidBlockHeight } } = await rpc.getLatestBlockhash().send();
```